client.go

```diff
@@ -19,71 +18,38 @@ import (
 const (
 	infoPath = "tunnels/info"
 
-	httpClientDefaultTimeout    = 60 * time.Second
-	httpClientTransportMaxConns = 100
-
 	// SCProtocol is name of the protocol used by Sauce Connect Proxy.
 	SCProtocol Protocol = "kgp"
 
 	// VPNProtocol is the protocol name used by IPSec Proxy.
 	VPNProtocol Protocol = "ipsec"
 )
 
-// defaultUserAgent is a default user-agent header for this client.
-var defaultUserAgent = fmt.Sprintf("TunnelRest/%s", Version)
-
-var httpClient = &http.Client{
-	// Timeout for requests made by this Client. The timeout includes conn
-	// time, any redirects, and reading the response body. The timer remains
-	// running after Get, Head, Post, or Do return and will interrupt reading
-	// of the Response.Body.
-	Timeout: httpClientDefaultTimeout,
-
-	Transport: &http.Transport{
-		MaxConnsPerHost:     httpClientTransportMaxConns,
-		MaxIdleConns:        httpClientTransportMaxConns,
-		MaxIdleConnsPerHost: httpClientTransportMaxConns,
-	},
-}
-
 // Client is the Sauce Connect Proxy REST API client. It allows you to create, query, and
 // terminate tunnels.
 type Client struct {
 	// BaseURL is REST API URL used for Sauce Connect queries.
 	BaseURL string
-	// GLobalBaseURL is REST API URL used for region-independent queries.
-	GlobalBaseURL string
+	// UserAgent specifies the user agent to be sent in the request.
```

Comment:
it's confusing to me but I THINK the logic before was:

- BaseURL  - regional REST API URL
- GlobalBaseURL - REST API for /updates queries (that are region-independent). Both URLs are the same, by default. Unless the client is used in staging or in dev. 

`GlobalBaseURL` is not really necessary (until we SL gets this global service deployed) so I don't mind removing it.